cmd/convox/install.go

```diff
@@ -199,6 +199,18 @@ func cmdInstall(c *cli.Context) {
 		stdcli.Error(fmt.Errorf("Convox is not currently supported in %s", region))
 	}
 
+	stackName := c.String("stack-name")
+	ecrRegex := "(?:[a-z0-9]+(?:[._-][a-z0-9]+)*/)*[a-z0-9]+(?:[._-][a-z0-9]+)*"
+	//regex found here:
+	//http://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_CreateRepository.html
+	rp := regexp.MustCompile(ecrRegex)
+	matchedStr := rp.FindString(stackName)
+	match := len(matchedStr) == len(stackName)
+
+	if !match {
+		stdcli.Error(fmt.Errorf("Stack name is invalid, must match %s", ecrRegex))
```

Comment:
instead of regex, can we say something more friendly "must contain alphanumeric characters and hyphens only"